### ISSUE TITLE
Add movement to starting position in `seq.scan()`

### ISSUE THREAD

**Issue by BrianJKoopman** [simonsobs/sorunlib#190]
Sometimes scans have accidentally been started in an incorrect location. Adding a starting position argument to `seq.scan()` and calling `acu.move_to()` within `seq.scan()` might mitigate the issue.

**Comment by newburgh** [comment]
I like this option; I definitely prefer to know the az/el the telescope is going to, which is solved if its an argument in the scan command. This requires a small modification to runlib and the scheduler?

**Comment by BrianJKoopman** [comment]
> I like this option; I definitely prefer to know the az/el the telescope is going to, which is solved if its an argument in the scan command. This requires a small modification to runlib and the scheduler?

Yup, implemented in #191 as an optional argument to `seq.scan()`. The scheduler can then start to use this whenever convenient.
